src/ai/backend/manager/actions/processor.py

```diff
@@ -28,8 +30,11 @@ async def _run(self, action: TAction) -> TActionResult:
         status: str = "unknown"
         description: str = "unknown"
         result: Optional[TActionResult] = None
+
+        task_id = uuid.uuid4()
```

Comment:
We said task_id earlier, but should we name it action_id?

Reply:
Resolved